tests/e2e/e2e_upgrade_test.go

```diff
@@ -49,7 +49,7 @@ var _ = OSMDescribe("Upgrade from latest",
 			// release branches, it should specify the most recent patch of the
 			// previous minor release. e.g. on the release-v1.0 branch, this
 			// should be "0.11".
-			i.Version = ">0.0.0-0"
+			i.Version = ">1.2"
```

Comment:
```suggestion
			i.Version = "1.1"
```

Just like this from the release-v1.1 branch: https://github.com/openservicemesh/osm/blob/fff4b0cc7c2d159cc21c0169eca73093fb81a3ec/tests/e2e/e2e_upgrade_test.go#L52